README.md

```diff
@@ -31,6 +31,59 @@ __Note:__ If you discovered any issue with the extension registry or have an ide
 
 5. `npm start`
 
+## Developing CSS  
+While the js, and html can be modified right in the repository, the sass stylesheets for brackets-registry exist in another repository - [brackets-site-sass](https://github.com/adobe/brackets-site-sass), and are included in as a submodule.
+
+This is because the same style sheets are shared by [brackets.io](https://github.com/adobe/brackets.io) as well.
+
+Refer to the section below for assistance on how to develop the css for brackets-registry
+
+- Requirements 
+    - Git command line tools — follow the setup instructions on [GitHub](https://help.github.com/articles/set-up-git) or download [here](http://git-scm.com/downloads)
+    - [NodeJS installed](https://nodejs.org/en/download/current/)
```

Comment:
Mention the node version. The current version might break some features in the future.